src/core/server/logging/layouts/json_layout.ts

```diff
@@ -46,20 +47,28 @@ export class JsonLayout implements Layout {
 
     return {
       message: error.message,
-      name: error.name,
-      stack: error.stack,
+      type: error.name,
+      stack_trace: error.stack,
     };
   }
 
   public format(record: LogRecord): string {
-    return JSON.stringify({
-      '@timestamp': moment(record.timestamp).format('YYYY-MM-DDTHH:mm:ss.SSSZ'),
-      context: record.context,
-      error: JsonLayout.errorToSerializableObject(record.error),
-      level: record.level.id.toUpperCase(),
-      message: record.message,
-      meta: record.meta,
-      pid: record.pid,
-    });
+    return JSON.stringify(
+      merge(
+        {
+          '@timestamp': moment(record.timestamp).format('YYYY-MM-DDTHH:mm:ss.SSSZ'),
+          message: record.message,
+          error: JsonLayout.errorToSerializableObject(record.error),
+          log: {
+            level: record.level.id.toUpperCase(),
+            logger: record.context,
+          },
+          process: {
+            pid: record.pid,
+          },
+        },
+        record.meta
```

Comment:
Similar to other comment, should we be placing `record.meta` under `labels`?

Reply:
meta might be named incorrectly - i tried to leave internal terminology as intact as possible, but it's closer to `fields` or something.  i could rename it

Reply:
Could you create an issue? I'm fine to rename it.